ui/apps/platform/src/routePaths.js

```diff
@@ -148,7 +151,16 @@ const vulnManagementPathToLabelMap = {
 export const basePathToLabelMap = {
     [dashboardPath]: 'Dashboard',
     [networkBasePath]: 'Network Graph',
-    [networkBasePathPF]: 'Network Graph-Patternfly',
+    [networkBasePathPF]: (
+        <Flex>
+            <FlexItem>New Network Graph</FlexItem>
+            <FlexItem>
+                <Badge style={{ backgroundColor: 'var(--pf-global--palette--gold-400)' }}>
```

Comment:
Not necessary yet tho, but I imagine we'll probably do something similar for later features. If necessary, we should make a component when we do